css/stanford_news_grid.css

```diff
@@ -7,12 +8,257 @@
+.view-stanford-news-grid.views-grid-two .views-row.views-row-414,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-416,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-418,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-420,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-422,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-424,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-426,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-428,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-430,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-432,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-434,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-436,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-438,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-440,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-442,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-444,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-446,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-448,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-450,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-452,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-454,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-456,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-458,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-460,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-462,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-464,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-466,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-468,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-470,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-472,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-474,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-476,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-478,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-480,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-482,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-484,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-486,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-488,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-490,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-492,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-494,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-496,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-498,
+.view-stanford-news-grid.views-grid-two .views-row.views-row-500 {
```

Comment:
Oh gosh that is a tonne of statements. 
I would like to introduce you to a wonderful shortcut for the 250 lines above.
`view-stanford-news-grid.views-grid-two .views-row:nth-child(2n)`